### PR TITLE
Set pymongo version 3.12.1

### PR DESCRIPTION
**Pull request by harrietc52**
Specify pymongo version 3.12.1, as 4.0 has breaking changes

### CODE REVIEW

File: Pipfile.lock [hunk 1]
```diff
@@ -778,19 +802,11 @@
...
             "hashes": [
-                "sha256:6c1246513ecd5ecd4528a0906f910e8f0f9c6b8ec72030dc9fd154dc1a6efd24",
-                "sha256:b8aa58f8cf793ffd8782d3d8cb19e66ef36f7aba4353eec859e74678b01b07a7"
+                "sha256:8e5643905bf20a308e25e4c1dd379117c09000bf8a82ebccc462cfb1b34a16b5",
+                "sha256:f71a09d7feba4a6b64ffd8e9d9bc60f9bf7d7e19fd0e04362acb1cfc2e3d98df"
             ],
             "index": "pypi",
-            "version": "==2.26.0"
-        },
-        "setuptools": {
```
Comment (KatyTaylor) [inline Pipfile.lock:787]: I guess we should add setuptools into the Pipfile to avoid losing it?

File: Pipfile.lock [hunk 2]
```diff
@@ -1321,14 +1338,6 @@
             "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'",
             "version": "==1.16.0"
         },
-        "toml": {
```
Comment (KatyTaylor) [inline Pipfile.lock:1324]: Same here - I guess we should add toml into the Pipfile to avoid losing it?

File: Pipfile.lock [hunk 3]
```diff
@@ -1267,11 +1284,11 @@
...
         "pytest": {
             "hashes": [
-                "sha256:131b36680866a76e6781d13f101efb86cf674ebb9762eb70d3082b6f29889e89",
-                "sha256:7310f8d27bc79ced999e760ca304d69f6ba6c6649c0b60fb0e04a4a77cacc134"
+                "sha256:8fc363e0b7407a9397e660ef81e1634e4504faaeb6ad1d2416da4c38d29a0f45",
+                "sha256:e1af71303d633af3376130b388e028342815cff74d2f3be4aeb22f3fd94325e6"
             ],
             "index": "pypi",
-            "version": "==6.2.5"
+            "version": "==7.0.0rc1"
```
Comment (KatyTaylor) [inline Pipfile.lock:1291]: This is a major version upgrade, might be risky?

File: Pipfile.lock (hunk 1 above)
Comment (harrietc52) [reply Pipfile.lock:787]: I've added it back to the Pipfile and ran `pipenv install setuptools` but it doesn't seem to be added it back into the .lock file. @emrojo any thoughts?

Comment (harrietc52) [reply Pipfile.lock:787]: Added it back manually

**Pull request closed without merging**